src/Randomizer.SMZ3.Tracking/Tracker.cs

```diff
@@ -783,7 +783,14 @@ public void ConnectToChat(string? userName, string? oauthToken, string? channel,
         {
             if (!string.IsNullOrEmpty(userName) && !string.IsNullOrEmpty(oauthToken))
             {
-                _chatClient.Connect(userName, oauthToken, channel ?? userName, id);
+                try
+                {
+                    _chatClient.Connect(userName, oauthToken, channel ?? userName, id);
+                }
+                catch (Exception e)
+                {
+                    Say(x => x.Chat.WhenDisconnected);
```

Comment:
Log the exception too, and maybe make the exception a specific type to avoid confusion.